runtime/binding-tcp/pom.xml

```diff
@@ -250,13 +249,22 @@
             <includes>
               <include>org.agrona:agrona</include>
               <include>io.aklivity.zilla:engine</include>
+              <include>jakarta.json:jakarta.json-api</include>
+              <include>jakarta.json.bind:jakarta.json.bind-api</include>
+              <include>org.eclipse:yasson</include>
+              <include>org.leadpony.justify</include>
+              <include>org.leadpony.joy:joy-yaml</include>
+              <include>org.leadpony.joy:joy-core</include>
+              <include>org.snakeyaml:snakeyaml-engine</include>
+              <include>com.ibm.icu:icu4j</include>
               <include>org.openjdk.jmh:jmh-core</include>
               <include>net.sf.jopt-simple:jopt-simple</include>
               <include>org.apache.commons:commons-math3</include>
               <include>commons-cli:commons-cli</include>
               <include>com.github.biboudis:jmh-profilers</include>
             </includes>
           </artifactSet>
+          <includeDependencies>true</includeDependencies>
```

Comment:
If this picks up dependencies transitively, can we trim the list above to no longer need to include any engine dependencies explicitly?

That would be better to implicitly keep the engine dependencies in sync.

Reply:
Also, suggest moving this configuration to top level `pom.xml` instead, with the rest of the default configuration for `shade` plugin as each project will need the same configuration.